distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCsw.java

```diff
@@ -0,0 +1,478 @@
+     *
+     * @param queryResult    - The result obtained from sending the query
+     * @param expectedValues - The values expected within the result
+     * @return
+     * @throws IOException
+     * @throws SAXException
+     * @throws ParserConfigurationException
+     * @throws XPathExpressionException
+     */
+    private boolean hasExpectedMetacardsReturned(String queryResult,
+            ExpectedResultPair[] expectedValues)
+            throws IOException, SAXException, ParserConfigurationException,
+            XPathExpressionException {
+        boolean testPassed = false;
+
+        for (int i = 0; i < expectedValues.length; i++) {
+            assertTrue("Metacard: " + expectedValues[i].value + " not found in result.",
+                    testPassed = queryResult.contains(metacardIds.get(expectedValues[i].value)));
+        }
+        return testPassed;
+    }
+
+    /**
+     * Ingests data, performs and validates the provided query returns the correct results.
+     *
+     * @param queryType - The query that is performed
+     * @throws XPathException
+     * @throws ParserConfigurationException
+     * @throws SAXException
+     * @throws IOException
+     */
+    private void performQueryAndValidateExpectedResults(String queryType)
+            throws XPathException, ParserConfigurationException, SAXException, IOException {
+        // Ingest
+        ingestMetacards();
+
+        String cswQuery = savedCswQueries.get(queryType);
+
+        String cswResponse = sendCswQuery(cswQuery);
+
+        hasExpectedResults(cswResponse, cswExpectedResults.get(queryType));
+    }
+
+    /**
+     * Validates that the query returned the expected result count.
+     *
+     * @param queryResult   - The result obtained from sending the query
+     * @param expectedValue - The values expected within the result
+     * @return
+     * @throws XPathException
+     * @throws IOException
+     * @throws SAXException
+     * @throws ParserConfigurationException
+     */
+    private boolean hasExpectedResultCount(String queryResult, ExpectedResultPair expectedValue)
+            throws XPathException, IOException, SAXException, ParserConfigurationException {
+
+        XPathExpression cswXPathExpression = getXPath().compile("//@numberOfRecordsMatched");
+
+        String originalCount = cswXPathExpression.evaluate(getDoc(queryResult));
+
+        return originalCount.equals(expectedValue.value);
+    }
+
+    private static XPath getXPath() {
+        return XPathFactory.newInstance()
+                .newXPath();
+    }
+
+    private static Document getDoc(String input)
+            throws ParserConfigurationException, IOException, SAXException {
+        return DocumentBuilderFactory.newInstance()
+                .newDocumentBuilder()
+                .parse(new ByteArrayInputStream(input.getBytes()));
+    }
+
+    public enum ResultType {
+        TITLE, COUNT
+    }
+
+    public static class ExpectedResultPair {
```

Comment:
Pretty sure this class no longer needs to be static, remnant of an old approach for the test.